pkg/config/aliases.go

```diff
@@ -0,0 +1,83 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
```

Comment:
I'm not a fan of duplicating our globals/constants to avoid editing the code using `config.XXX`. Updating the code base would produce a much bigger PR but would improve the final result in my opinion.

We could do this in another PRs if it's easier.

Reply:
I agree, I think every time I've seen this kind of change we just edited the calls all over the repo to keep a clean state (ie. avoid having re-exports of a module inside another one)

Reply:
Perfect, could you create a card in your backlog so we can track this ?

Reply:
Created https://datadoghq.atlassian.net/browse/OTEL-1134 to track this